guides/common/modules/proc_overriding-ansible-variables.adoc

```diff
@@ -12,42 +15,56 @@ To ensure that the variable that you override follows the correct order of prece
 .Prerequisite
 
 * You must have Ansible variables in {Project}.
-
-** To import Ansible variables, see {ConfiguringAnsibleDocURL}importing-Ansible-variables_ansible[Importing Ansible Variables]
-
-** To create Ansible variables, see {ConfiguringAnsibleDocURL}creating-Ansible-variables_ansible[Creating Ansible Variables]
-
-The following procedure makes reference to hosts and host groups.
-For more information about hosts and host groups, see the {ManagingHostsDocURL}[Managing Hosts] guide.
+For more information, see xref:importing-Ansible-roles_{context}[]
 
 .Procedure
 
 . In the {ProjectWebUI}, navigate to *Configure* > *Variables*.
 . Select the Ansible variable that you want to override and manage with {Project}.
 . Navigate to the *Default Behavior* area, and select the *Override* check box.
```

Comment:
```suggestion
. Navigate to the *Default Behavior* area and select the *Override* check box.
```

I think "navigate" is wrong here; it's more "scroll down to ...". Maybe say "In the *Default Behavior* area, select ..."